acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerRunner.java

```diff
@@ -47,6 +52,8 @@
   private final Path dataPath;
   private final Properties portsProperties;
 
+  private EmbeddedPostgres slashingDatabase;
```

Comment:
can this can be private

Reply:
it is private - but cannot be final, as it doesn't get created until "start"